src/client/app/component/HomePageHeader/index.js

```diff
@@ -39,6 +39,13 @@ class HomePageHeader extends React.Component {
   redirectTopage(path) {
     this.props.history.push(path);
   }
+  handleMenuBtnClick(e) {
+    if (e.key === "1") {
+      this.logOut();
+    } else if (e.key === "2") {
+      this.redirectTopage.bind(this, '/create-business')();
```

Comment:
use `call` instead of `bind`

**What's the difference between `bind` and `call`?**
`bind` will not call your function, *the function will not run*, it will just return a function and that function will have the value of `this` and the arguments preset.

`call` will call your function, *the function will run*, with the given value of `this` and the arguments.

**What's the difference between `apply` and `call`?**
They do the same but `call` accepts the arguments individually while `apply` accepts them as an array.

Read more about [bind](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind), [call](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) and [apply](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply) on MDN.